### ISSUE TITLE
Android Dynamic Feature Support

### ISSUE THREAD

**Issue by princeparadoxes** [MarathonLabs/marathon#690]
marathon supports testing multiple modules
https://marathonlabs.github.io/marathon/ven/android.html#multi-module-testing

But, if use that case for dynamic feature adb throwing error
`Failure [INSTALL_FAILED_INVALID_APK: Full install must include a base package]`

Cause of error:
marathon invoke `adb install` for each apk. Dynamic feature needs `adb install-multiple` with some apk.

List of adb command for install and run test of dynamic feature:
```
adb uninstall com.dynamic.feature
adb uninstall com.main

adb install-multiple app/build/outputs/apk/debug/main.apk modules/dynamic/build/outputs/apk/debug/dynamic-feature.apk 

adb install modules/dynamic/build/outputs/apk/androidTest/debug/dynamic-feature-androidTest.apk

adb shell am instrument -w -r --no-window-animation -e class com.dynamic.feature.DynamicTest#someTest -e com.dynamic.feature/com.dynamic.feature.SomeUiTestRunner
```

**Comment by Malinskiy** [comment]
Released in 0.8.0

**Comment by pekspadatahun-ai** [comment]
INSTALL_FAILED_INVALID_APK: Full install must include a base package
